screenshots/README.md

```diff
@@ -3,10 +3,10 @@ Screenshots
 
 These screenshots are generated by running the selenium tests, e.g. using
 
-    npm test -- -t Screenshot
+    npm run test:e2e
 
 That will override screenshots, but not delete files that are no longer
-generated by the test; in that case, run `rm -f screenshots/*.png` before
+generated by the test; in that case, run `rm -f screenshots/**/*.png` before
```

Comment:
idea: what about having `XX-<test-name>-{desktop|mobile}.png`, all in the same directory? I think it's nice to be able to compare desktop and mobile screenshots side by side (i.e. without having to open two different directories).

Reply:
sure, why not